components/footer/footer.tsx

```diff
@@ -0,0 +1,123 @@
+import Image from 'next/image';
+import styles from '../../styles/Home.module.css';
+import { classnames } from 'tailwindcss-classnames';
+
+const Footer = (): JSX.Element => {
+  return (
+    <footer className={classnames('bg-charcoal', 'flex', 'text-white', 'justify-center', 'py-12')}>
+      <div className={classnames('flex', 'container', 'flex-col', 'px-4', 'sm:px-0')}>
+        <div className={classnames('flex', 'flex-col', 'mb-4')}>
+          <a href="/help">
+            <h2>Need help?</h2>
+          </a>
+          <a href="https://developers.sitecore.com" target="_blank" rel="noopener noreferrer">
+            Powered by{' '}
+            <span className={classnames('ml-2')}>
+              <Image src="/sitecore-white.svg" alt="Sitecore Logo" width={72} height={16} />
+            </span>
+          </a>
```

Comment:
Next Image is kind of a butt. Was just gonna say add some classes to the anchor to make the sitecore image vertically centered... but again Next Image doesn't place nice with that.

```
<a className="flex items-center" href="https://developers.sitecore.com" target="_blank" rel="noopener noreferrer">
  <span className="mr-2">Powered by{' '}</span>
  <Image src="/sitecore-white.svg" alt="Sitecore Logo" width={72} height={16} />
</a>
```

Here's the absolutely crap workaround I found

Reply:
Ty Brian -- will add when I get a chance after investigating the global css issue.